packages/react/src/components/RadioCardGroup/index.tsx

```diff
@@ -0,0 +1,183 @@
+import classNames from 'classnames';
+import Icon, { IconType } from '../Icon';
+import Card, { CardContent } from '../Card';
+
+export interface RadioItem extends React.InputHTMLAttributes<HTMLInputElement> {
+  label: React.ReactNode;
+  cardImgSrc: string;
+  cardIcon: IconType;
+  value?: string;
+}
+
+export interface RadioCardGroupProps {
+  name?: string;
+  className?: string;
+  radios: RadioItem[];
+  defaultValue?: string;
+  value?: any;
+  onChange: (radio: RadioItem, input: HTMLElement) => void;
+}
+
+const RadioCardGroup = ({
+  name,
+  radios,
+  defaultValue,
+  value,
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  onChange = () => {},
+  className,
+  ...other
+}: RadioCardGroupProps) => {
+  const [currentValue, setCurrentValue] = useState<string | null>(
+    value || defaultValue || null
+  );
+  const [focusIndex, setFocusIndex] = useState<number | null>(null);
+  const inputs = useRef<HTMLInputElement[]>([]);
+  const handleChange = (value: any) => setCurrentValue(value);
+  const onRadioFocus = (index: number) => setFocusIndex(index);
+  const onRadioBlur = () => setFocusIndex(null);
+  const onRadioClick = (index: number) => {
+    const radio = inputs.current?.[index];
+
+    if (!radio) {
+      return;
+    }
+
+    radio.click();
+    radio.focus();
+  };
+
+  useEffect(() => {
+    if (typeof value === 'undefined') {
+      return;
+    }
+
+    setCurrentValue(value);
+  }, [value]);
+
+  const radioButtons = radios.map((radio, index) => {
+    const {
+      label,
+      disabled,
+      value: radioValue,
+      cardImgSrc,
+      cardIcon,
+      id,
+      className,
+      ...other
+    } = radio;
+    const isChecked = currentValue === radioValue;
+    const isFocused = focusIndex === index;
+
+    return (
+      <div className={classNames('Radio')} key={id}>
+        <Card
+          variant="simple"
+          className={classNames('Radio_card Radio__overlay', {
+            'Radio__overlay--focused': isFocused,
+            'Radio__overlay--checked': isChecked,
+            'Radio__overlay--disabled': disabled
+          })}
+          type={isChecked ? 'radio-checked' : 'radio-unchecked'}
+          onClick={() => onRadioClick(index)}
+        >
+          <input
+            type="radio"
+            name={name}
+            value={radioValue}
+            id={id}
+            ref={input => {
+              if (!input) {
+                return;
+              }
+
+              inputs.current.push(input);
+            }}
+            onFocus={() => onRadioFocus(index)}
+            onBlur={() => onRadioBlur()}
+            onChange={() => {
+              handleChange(radioValue);
+              onChange(radio, inputs.current?.[index]);
+            }}
+            disabled={disabled}
+            checked={isChecked}
+            {...other}
+          />
+          <CardContent>
+            <div className={classNames('Radio_card_icon_div')}>
```

Comment:
I'm not sure this wrapper is necessary. We should be able to position the `Icon` with absolute styles:

```css
.RadioCardGroup {
  position: relative;
}

.RadioCardGroup .Icon {
  position: absolute;
  top: n; /*whatever the x value needs to be */
  right: n; /* whatever the y value needs to be */
}
```

Reply:
I found I need to have the wrapper to stop from this occurring when clicking a card, this way the div is always set with a min-height


before: 
<img width="313" alt="image" src="https://user-images.githubusercontent.com/41127686/172412649-381b9d2e-2b21-42e4-aa14-dc7d78bf2105.png">
after:
<img width="309" alt="image" src="https://user-images.githubusercontent.com/41127686/172412695-6f0c69a1-9e28-43e6-8220-a91a52c557fd.png">